src/components/KTextArea/KTextArea.vue

```diff
@@ -268,24 +309,47 @@ $kTextAreaPaddingY: var(--kui-space-40, $kui-space-40); // corresponds to mixin,
   .help-text {
     @include inputHelpText;
 
-    // fixing mixed-decls deprecation: https://sass-lang.com/d/mixed-decls
-    // stylelint-disable-next-line no-duplicate-selectors
-    & {
-      // reset default margin from browser
-      margin: 0;
-      margin-top: var(--kui-space-40, $kui-space-40) !important; // need important to override some overrides of default p margin in other components
+    // reset default margin from browser
+    margin: 0;
+    margin-top: var(--kui-space-40, $kui-space-40) !important; // need important to override some overrides of default p margin in other components
+  }
+
+  .input-wrapper {
```

Comment:
nit: can we rename to `input-textarea-wrapper` to match the naming below?

Reply:
renamed